run/archiveloop

```diff
@@ -295,6 +295,16 @@ function archive_clips () {
   connect_usb_drives_to_host
 }
 
+function check_if_usb_gadget_is_mounted () {
+    if [ ! -d /devices/platform/soc/20980000.usb/gadget/lun0 ]
```

Comment:
I think you mean '/sys/devices/platform/soc/20980000.usb/gadget/lun0 ?
Because /devices does not exist on my install.

Reply:
Ah yes. It is /sys/devices. I tried this with 2 different pi's and 4 sd card and all of them stopped working between 16-20 hours of inactivity (despite a powerbank backup). This fix seemed to have helped me. I can test for one more week if you want me to.